documentation/_puppetdb_nav.html

```diff
@@ -3,114 +3,114 @@
-      <li><a href="/puppetdb/{{ puppetdb_version }}/puppetdb-faq.html">Frequently Asked Questions</a></li>
-      <li><a href="/puppetdb/{{ puppetdb_version }}/release_notes.html">Release Notes</a></li>
-      <li><a href="/puppetdb/{{ puppetdb_version }}/versioning_policy.html">Versioning Policy</a></li>
-      <li><a href="/puppetdb/{{ puppetdb_version }}/known_issues.html">Known Issues</a></li>
-      <li><a href="/puppetdb/{{ puppetdb_version }}/community_add_ons.html">Community Add-ons</a></li>
+      <li><a href="/puppetdb/{{ puppetdb_version }}/index.html">Overview and requirements</a></li>
+      <li><a href="/puppetdb/{{ puppetdb_version }}/puppetdb-faq.html">Frequently asked questions</a></li>
+      <li><a href="/puppetdb/{{ puppetdb_version }}/release_notes.html">Release notes</a></li>
+      <li><a href="/puppetdb/{{ puppetdb_version }}/versioning_policy.html">Versioning policy</a></li>
+      <li><a href="/puppetdb/{{ puppetdb_version }}/known_issues.html">Known issues</a></li>
+      <li><a href="/puppetdb/{{ puppetdb_version }}/community_add_ons.html">Community add-ons</a></li>
     </ul>
   </li>
   <li><strong>Installation</strong>
     <ul>
-      <li><a href="/puppetdb/{{ puppetdb_version }}/migrate.html">Migrating Existing Data</a></li>
-      <li><a href="/puppetdb/{{ puppetdb_version }}/install_via_module.html">Installing via Puppet Module</a></li>
-      <li><a href="/puppetdb/{{ puppetdb_version }}/install_from_packages.html">Installing From Packages</a></li>
-      <li><a href="/puppetdb/{{ puppetdb_version }}/install_from_source.html">Installing from Source</a></li>
+      <li><a href="/puppetdb/{{ puppetdb_version }}/migrate.html">Migrating data</a></li>
+      <li><a href="/puppetdb/{{ puppetdb_version }}/install_via_module.html">Installing via Puppet module</a></li>
+      <li><a href="/puppetdb/{{ puppetdb_version }}/install_from_packages.html">Installing from packages</a></li>
+      <li><a href="/puppetdb/{{ puppetdb_version }}/install_from_source.html">Installing from source</a></li>
       <li><a href="/puppetdb/{{ puppetdb_version }}/upgrade.html">Upgrading PuppetDB</a></li>
-      <li><a href="/puppetdb/{{ puppetdb_version }}/connect_puppet_master.html">Connecting Puppet Masters</a></li>
-      <li><a href="/puppetdb/{{ puppetdb_version }}/connect_puppet_apply.html">Connecting Standalone Puppet</a></li>
+      <li><a href="/puppetdb/{{ puppetdb_version }}/connect_puppet_master.html">Connecting Puppet masters</a></li>
+      <li><a href="/puppetdb/{{ puppetdb_version }}/connect_puppet_apply.html">Connecting standalone Puppet nodes</a></li>
     </ul>
   </li>
   <li><strong>Configuration</strong>
     <ul>
       <li><a href="/puppetdb/{{ puppetdb_version }}/configure.html">Configuring PuppetDB</a></li>
-      <li><a href="/puppetdb/{{ puppetdb_version }}/puppetdb_connection.html">puppetdb.conf: Configuring a Puppet/PuppetDB Connection</a></li>
```

Comment:
I also suggest preserving the filename, for exactly that reason. Saves the reader time.